script/test-certs/ca.cert.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is this file needed? If we are not configuring the client to verify we do not need to pass this file during helm init. I believe.

Reply:
we are using `verify` for `helm init`:

```
  --tiller-tls-verify \
   --tls-ca-cert ${CERTS_DIR}/ca.cert.pem
```

I could remove that as well

Reply:
Well, the issue is that it can be confusing since it is not actually being used.